packages/puppeteer-core/src/api/BrowserContext.ts

```diff
@@ -198,31 +190,48 @@ export class BrowserContext extends EventEmitter<BrowserContextEvents> {
   }
 
   /**
-   * Creates a new page in the browser context.
+   * Creates a new {@link Page | page} in this
+   * {@link BrowserContext | browser context}.
    */
-  newPage(): Promise<Page> {
-    throw new Error('Not implemented');
-  }
+  abstract newPage(): Promise<Page>;
 
   /**
-   * The browser this browser context belongs to.
+   * Gets the {@link Browser | browser} associated with this
+   * {@link BrowserContext | browser context}.
    */
-  browser(): Browser {
-    throw new Error('Not implemented');
-  }
+  abstract browser(): Browser;
 
   /**
-   * Closes the browser context. All the targets that belong to the browser context
-   * will be closed.
+   * Closes this {@link BrowserContext | browser context} and all associated
+   * {@link Page | pages}.
    *
-   * @remarks
-   * Only incognito browser contexts can be closed.
+   * @remarks The
+   * {@link Browser.defaultBrowserContext | default browser context} cannot be
+   * closed.
    */
-  close(): Promise<void> {
-    throw new Error('Not implemented');
+  abstract close(): Promise<void>;
+
+  /**
+   * Whether this {@link BrowserContext | browser context} is closed.
+   */
+  get closed(): boolean {
```

Comment:
is it internal or why I don't find a docs update? Is it a feat or refactor?

Reply:
https://github.com/puppeteer/puppeteer/pull/10928#discussion_r1328957022